### PR TITLE
CreateTableHandler return table info

### PR DESCRIPTION
**Pull request by romanbracinik**
Pridam v response create table aby to vratilo TableInfo upravil som aj BQ driver este poslem PR do connection 

Uprava v BigQuery: https://github.com/keboola/php-storage-driver-teradata/pull/42

### CODE REVIEW

File: src/Handler/Table/Create/CreateTableHandler.php [hunk 1]
```diff
@@ -83,11 +85,18 @@ public function __invoke(
...
+                $command->getTableName()
+            );
         } finally {
             if (isset($db)) {
                 $db->close();
             }
         }
-        return null;
+
+        return TableReflectionResponseTransformer::transformTableReflectionToResponse($databaseName, $ref);
```
Comment (martinjunger) [inline src/Handler/Table/Create/CreateTableHandler.php:100]: no, podle me by to melo vracet spis `ObjectInfoResponse` + `ObjectType::TABLE`, jako je to v https://github.com/keboola/php-storage-driver-teradata/blob/1f1ef3a51893de90e4f6859a40db25100d148ea8/src/Handler/Info/ObjectInfoHandler.php#L158-L158

Ted to vraci rovnou `TableInfo`, coz je matouci, pac to neni `*Response` a chova se to odlisne...

File: tests/functional/UseCase/Table/PreviewTableTest.php [hunk 2]
```diff
@@ -470,12 +470,11 @@ private function createTable(string $databaseName, string $tableName, array $str
             ->setColumns($columns)
             ->setPrimaryKeysNames($primaryKeysNames);
 
-        $createTableResponse = $createTableHandler(
+        $createTableHandler(
             $this->projectCredentials,
             $createTableCommand,
             []
         );
-        $this->assertNull($createTableResponse);
```
Comment (martinjunger) [inline tests/functional/UseCase/Table/PreviewTableTest.php:478]: Tady by potom melo fungovat:

```php
        $this->assertInstanceOf(ObjectInfoResponse::class, $createTableResponse);
        $this->assertSame(ObjectType::TABLE, $createTableResponse->getObjectType());
```

**Review by martinjunger — CHANGES_REQUESTED** [review]
Mela by se vracet `ObjectInfoResponse`, nikoliv jen `TableInfo`, viz komentare...

File: src/Handler/Table/Create/CreateTableHandler.php (hunk 1 above)
Comment (romanbracinik) [reply src/Handler/Table/Create/CreateTableHandler.php]: Jo super diky :)

File: tests/functional/UseCase/Table/PreviewTableTest.php (hunk 2 above)
Comment (romanbracinik) [reply tests/functional/UseCase/Table/PreviewTableTest.php]: Upravil som

**Review by martinjunger — APPROVED** [review]
👍

**Pull request merged**